source/compiler/sc4.c

```diff
@@ -1245,7 +1245,8 @@ SC_FUNC void inc(value *lval)
 {
   symbol *sym;
 
-  sym=lval->sym;
+  if ((sym=lval->sym)!=NULL)
```

Comment:
What about:

```
sym=lval->sym;
if (sym!=NULL)
    markusage(sym,uWRITTEN);
```

Reply:
I only followed the existing coding style, as `sym` is assigned a value inside of `if` control expressions in many other cases, like this:
https://github.com/pawn-lang/compiler/blob/29770ae31b0a47ae4767d2fdfb979643e2309e4b/source/compiler/sc1.c#L2312-L2319
or this:
https://github.com/pawn-lang/compiler/blob/29770ae31b0a47ae4767d2fdfb979643e2309e4b/source/compiler/sc3.c#L2643
But I can fix this, if such practice is undesirable.